packages/tools/client-debugger/client-debugger-view/src/components/index.ts

```diff
@@ -10,6 +10,7 @@ export * from "./client-data-views";
 
 export * from "./AudienceView";
 export * from "./ContainerDevtoolsView";
+export * from "./ContainerHistoryLog";
```

Comment:
Seems like all the exports from the module are only used in ContaineryHistoryView so I think this export is unnecessary.

Reply:
Fair point!
```suggestion
```